app/models/available_date.rb

```diff
@@ -0,0 +1,8 @@
+class AvailableDate
+  include DataMapper::Resource
+
+  property :id, Serial
+  property :available_date, Date
```

Comment:
This is a bit of a smell. If a model has a property with the name of the model, it suggests that there isn't really a model here at all.